Main/Source/item.cpp

```diff
@@ -2086,6 +2086,20 @@ void item::SendMemorizedUpdateRequest() const
       }
 }
 
+void item::AddContainerPostFix(festring& String) const
+{
+  if(GetSecondaryMaterial()){
+    float fRatio = GetSecondaryMaterial()->GetVolume()/(float)GetDefaultSecondaryVolume();
+    const char* c="full of";
+    if     (fRatio<=0.10)c="with just a little bit of";
+    if     (fRatio<=0.25)c="with a bit of";
+    else if(fRatio<=0.50)c="with some";
+    else if(fRatio<=0.75)c="with"; //TODO any better phrasing for this?
+    else if(fRatio<=0.90)c="almost full of";
```

Comment:
Shouldn't here be <1.00 so taht bottles can be "full of"?

Reply:
they will show as "full of" whenever > 0.90

I thought that 0.999 could still show as "almost full of" and then we try to add more spider poison just to get 1ml out of it and lose the remaining.
In the other hand, 10% of the liquid will not make much diff and if we look at the bottle that is 90% (irl) it may look like full.

Should I just documented why it is 0.90 or better be sure we fill the last 1ml? xD

Reply:
Oh, no, I just misread the ratio check and thought anything OVER 0.90 would be "almost full of". My bad.

I think it's OK as is, no one will be angry that "full of" can also be "0.904658753 full". :P

Reply:
if it looks like it is full... than it is full! xD